pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "weheat"
-version = "2024.08.10"
+version = "2024.08.30"
```

Comment:
Is it possilble to define version string in just one place instead of pyproject and setup.py?